terser.config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compress": {
+    "arguments": true,
+    "booleans_as_integers": true,
+    "keep_fargs": false,
+    "module": true,
+    "passes": 3,
+    "unsafe_arrows": true,
+    "unsafe_math": true,
+    "unsafe_undefined": true,
+    "unsafe_proto": true,
+    "unsafe_Function": true,
+    "unsafe_comps": true,
+    "hoist_funs": true,
+    "ecma": 6
+  }
+}
```

Comment:
Not sure about these options, this could potentially break the card in older browsers

Reply:
I haven't tested it in older browsers, more tests would be needed.

Reply:
I think it should be fine as terser is added as the last step (after babel). But worth to double check on some older browsers/devices (like tablets - there is a lot of people using old tablets as wall-panels).

Reply:
Karl is right I think. These two options can break the code for devices not supporting ES6: `unsafe_arrows`, `ecma: 6`

Reply:
Isn't it only IE11 that doesn't support ES6?

Reply:
No, people often use old tablets as wall panels so in theory this could break the card for them but...

as I said in a different comment - currently transpiler converts all async functions in generator functions which means the card is broken anyway as older browsers don't support them

Reply:
Might be worth looking at publishing two files that users can switch between. One for completely modern browsers and one for legacy ones?